server/plugin/api.go

```diff
@@ -713,13 +714,14 @@ func (p *Plugin) getReviewsData(c *UserContext) []*github.Issue {
 	result, _, err := githubClient.Search.Issues(c.Ctx, query, &github.SearchOptions{})
 	if err != nil {
 		c.Log.WithError(err).With(logger.LogContext{"query": query}).Warnf("Failed to search for review")
-		return nil
+		return issue
```

Comment:
This var is not initialized, only declared, so its value is `nil` here. I suggest just returning it like this, and not have a variable declared above since it's only referenced in one place here. Same with other instances of this in this commit

```suggestion
		return []*github.Issue{}
```